GoogleForCreators/web-stories-wp
# Shopping: Products Sidebar Tab

## Context

Establish new products sidebar tab 



## Summary

- Adds new shopping  icon 
- Added new title & description
- Added new search form
- Added new search results list with images, title, and price



## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions

CI test issue fixes.






This PR can be tested by following these steps:

1. Ensure shopping is enabled
2. Visit the shopping tab
3. Find a product via the search box
4. Use the `+` button to add the product
5. Search again ... not the already added product has a ✅ icon
5. Hover over the checked product and remove it via the `X` button


## Reviews

### Does this PR have a security-related impact?

No



### Does this PR change what data or activity we track or use?
No



### Does this PR have a legal-related impact?
No



## Checklist




---



Fixes #11155
